dex/networks/zec/params.go

```diff
@@ -0,0 +1,86 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org
+
+package zec
+
+import (
+	"decred.org/dcrdex/dex"
+	"decred.org/dcrdex/dex/networks/btc"
+	"github.com/btcsuite/btcd/chaincfg"
+	dcrchaincfg "github.com/decred/dcrd/chaincfg/v3"
+)
+
+const (
+	// MinimumTxOverhead
+	// 4 header + 4 nVersionGroup + 1 varint input count + 1 varint output count
+	// + 4 lockTime + 4 nExpiryHeight + 8 valueBalanceSapling + 1 varint nSpendsSapling
+	// + 1 varint nOutputsSapling + 1 varint nJoinSplit
+	MinimumTxOverhead = 29
+
+	InitTxSizeBase = MinimumTxOverhead + btc.P2PKHOutputSize + btc.P2SHOutputSize // 29 + 34 + 32 = 95
+	InitTxSize     = InitTxSizeBase + btc.RedeemP2PKHInputSize                    // 95 + 149 = 244
+)
+
+var (
+	UnitInfo = dex.UnitInfo{
+		AtomicUnit: "Sats",
```

Comment:
Is it a "Zat"/zatoshi? I see stuff like `amountZat`, `inZat`, `valueZat` in RPCs and the zcashd code

https://github.com/zcash/zcash/commit/1805466d5c555778017bdb165688fd699bcbcdbb